packages-content-model/roosterjs-content-model-dom/lib/modelToDom/handlers/handleEntity.ts

```diff
@@ -23,7 +24,23 @@ export const handleEntityBlock: ContentModelBlockHandler<ContentModelEntity> = (
 
     applyFormat(wrapper, context.formatAppliers.entity, entityFormat, context);
 
-    refNode = reuseCachedElement(parent, wrapper, refNode);
+    const isCursorAroundEntity =
+        context.addDelimiterForEntity &&
+        wrapper.style.display == 'inline-block' &&
+        wrapper.style.width == '100%';
+    const isContained = wrapper.parentElement?.classList.contains('blockEntityContainer');
```

Comment:
Make "blockEntityContainer" a constant.

Shall we use a more specific class name? Such as _E_BlockContainer, in order to reduce the chance to be conflict with other class names

Reply:
Sounds good. Changed the className